types/stripe/index.d.ts

```diff
@@ -1305,12 +1305,12 @@ declare namespace Stripe {
             /**
              * ID of the payment method used in this charge.
              */
-            payment_method: string;
+            payment_method: string | null;
```

Comment:
When would this and the next one be null? If using sources instead of payment methods?

Reply:
Yes, that's exactly right.